<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dly_Map2RegFIFO</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="Dly_Map2RegFIFO">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dly_inReqFIFO</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="Dly_inReqFIFO"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Dly_onPick</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="Dly_onPick">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Dly_inConv</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="Dly_inConv">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Dly_Conv2SndFIFO</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="Dly_Conv2SndFIFO">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dly_inSndFIFO</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="Dly_inSndFIFO"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dly_SndPipe</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="Dly_SndPipe">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Dly_inSwitch</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="Dly_inSwitch"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Dly_Switch2NextPE</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="Dly_Switch2NextPE">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Packet Point Count</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="PacketPointCount">
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>ReqFIFO Max Size</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="ReqFIFO_MaxSize">
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Picker Full Bandwidth</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpinBox" name="Picker_FullBandwidth">
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>ConvsFIFO Max Size</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QSpinBox" name="ConvFIFO_MaxSIze">
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Switch Full Bandwidth</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QSpinBox" name="Switch_FullBandwidth">
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background: url(:/images/PE)</string>
      </property>
      <widget class="QLabel" name="RegFIFO">
       <property name="geometry">
        <rect>
         <x>363</x>
         <y>53</y>
         <width>72</width>
         <height>112</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(90, 154, 215);
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>ReqFIFO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="DatLatch">
       <property name="geometry">
        <rect>
         <x>492</x>
         <y>54</y>
         <width>73</width>
         <height>113</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(90, 154, 215);
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>ReqDatLatch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="SndFIFO">
       <property name="geometry">
        <rect>
         <x>363</x>
         <y>378</y>
         <width>72</width>
         <height>112</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(90, 154, 215);
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>SndFIFO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="SwitchFIFO">
       <property name="geometry">
        <rect>
         <x>296</x>
         <y>612</y>
         <width>17</width>
         <height>59</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(2, 31, 99);
color:white;
font: 7px;
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>FIFO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="SndPipe">
       <property name="geometry">
        <rect>
         <x>309</x>
         <y>542</y>
         <width>40</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(89, 154, 214);
color:white;
font: 7px;
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>SndPipe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Conv2">
       <property name="geometry">
        <rect>
         <x>305</x>
         <y>255</y>
         <width>196</width>
         <height>76</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(56, 86, 36);
color: white;
border-radius:3px;</string>
       </property>
       <property name="text">
        <string>Conv1,Conv2,Conv3...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>320</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
